docs/project-structure/genezio-configuration-file.md

```diff
@@ -144,8 +180,23 @@ The functions that will be deployed to the cloud. This field can be omitted if t
 - `type`: `string` **Optional**
 
   The type of the function. The default value is `aws`.
+### Backend with `functions` deployment
+```yaml
```

Comment:
Please add comments for this `genezio.yaml` snippet, too.

You can use this one [1] as an example.

[1]  https://genezio.com/docs/tutorials/how-to-deploy-an-express-app/#3-create-the-genezio-configuration-file